ciri/conf.c

```diff
@@ -204,13 +218,7 @@ static retcode_t set_conf_value(iota_ciri_conf_t* const ciri_conf, iota_consensu
                              HASH_LENGTH_TRYTE, HASH_LENGTH_TRYTE);
       break;
     case CONF_SNAPSHOT_SIGNATURE_SKIP_VALIDATION:  // --snapshot-signature-skip-validation
-      if (strcmp(value, "true") == 0) {
-        consensus_conf->snapshot_signature_skip_validation = true;
-      } else if (strcmp(value, "false") == 0) {
-        consensus_conf->snapshot_signature_skip_validation = false;
-      } else {
-        return RC_CIRI_CONF_INVALID_ARGUMENTS;
-      }
+      ret = get_true_false(value, &consensus_conf->snapshot_signature_skip_validation);
```

Comment:
the `ret` will be overwritten in the other case. say case CONF_TIPS_SOLIDIFIER_ENABLED set `ret` to RC_CIRI_CONF_INVALID_ARGUMENTS, but it could be set to RC_OK in this case.

Reply:
Parameters are processed one by one by `set_conf_value` so there is no risk of overriding `ret`.